client/signup/steps/design-picker/index.jsx

```diff
@@ -49,24 +72,35 @@ class DesignPickerStep extends Component {
 		let designs = undefined;
 
 		if ( this.props.showOnlyThemes ) {
-			// Only offering designs that are also available as themes. This means excluding
-			// designs where the `template` has a layout that's different from what the theme's
-			// default Headstart annotation provides.
-			designs = getAvailableDesigns().featured.filter(
-				( { features, template, theme } ) => theme === template && ! features.includes( 'anchorfm' )
-			);
+			// Should probably be pulled out into a utility to format theme response as design picker design
+			// taxonomies.theme_subject probably maps to category
+			designs = this.props.themes
```

Comment:
We can create a selector for this case and then we don't need to do filtering and mapping all the time. But we can do it in another PR.

Reply:
I've updated the comment to reflect what I actually think we should do, which is perhaps move the fetching code and filtering code into the `<DesignPicker>` itself. In fact as the idea of "design" becomes less important, I don't think we'll need this intermediary design format, and the `<DesignPicker>` can just start consuming the API response format directly.